tools/rsdl/rapid-cli/rapid.rdm.transformation.tests/data/namespaces/service.csdl.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<edmx:Edmx Version="4.0" xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx">
+  <edmx:Reference Uri="http://unknown.com">
```

Comment:
```suggestion
  <edmx:Reference Uri="import1">
```

As default assume URL structure to follow file structure.

Reply:
Yes, the http://unknown.com is only a placeholder and needs to be fixed.


But we also need to talk about the include statement again. At the moment it is 
```ABNF
  include :=  "include" quoted-string "as" identifier
```
That means we are currently opening the file and reading the namespace from that file.
I do not think that this is reasonable if the file is not local like in the case of the Annotation Term files.

So I would (in addition to above) propose to change the include statement to be

```ABNF
  include :=  "include" identifier from quoted-string "as" identifier
```

This also helps when including CSDL files which potentially define two namespaces.

Reply:
Why did you close the PR?

Reply:
I am not able to complete this in the near future. 
The .NET OData library is currently not able to add (and write) annotations on schema, include or reference.